README.md

```diff
@@ -32,53 +32,114 @@
 
 The building blocks of our wildfire detection & monitoring API.
 
+## Quick Tour
 
-## Getting started
+### Running/stopping the service
 
-### Prerequisites
+You can run the API containers using this command:
 
-- Python 3.7 (or more recent)
-- [pip](https://pip.pypa.io/en/stable/)
+```shell
+make run
+```
 
-### Installation
+You can now navigate to `http://localhost:8080/docs` to interact with the API (or do it through HTTP requests) and explore the documentation.
 
-You can clone and install the project dependencies as follows:
+![Swagger](https://github.com/pyronear/pyro-api/releases/download/v0.1.2/swagger_interface.png)
 
+In order to stop the service, run:
 ```shell
-git clone https://github.com/pyronear/pyro-api.git
+make stop
 ```
 
-## Usage
+### How is the database organized
+
+The back-end core feature is to interact with the metadata tables. For the service to be useful for wildfire detection, multiple tables/object types are introduced and described as follows:
+
+#### Access-related tables
+
+- Groups: defines collections of credentials that share a similar scope (e.g. you won't be able to access the same data as the local firefighters).
+- Accesses: stores the hashed credentials and access level for users & devices.
+- Users: actual humans registered in the database.
+- Devices: the registered cameras.
+
+#### Setup-specific tables
+
+- Sites: specific locations (firefighter watchtowers, fire stations, etc.).
+- Installations: association linking a device & a site over a given timespan.
+
+#### Core detection worklow tables
+
+- Events: wildfire events.
+- Media: metadata of a picture and its storage bucket key.
+- Alerts: association of a picture, a device, and an event.
+
+#### Advanced tables
+
+- Webhooks: advanced mechanisms to introduce callbacks on specific routes.
+
+
+### What is the full detection workflow through the API
+
+The API has been designed to provide, for each wildfire detection, the alert metadata:
+- timestamp
+- the picture that was used for detection
+- the location is was taken at, and the direction it was taken from
 
-If you wish to deploy this project on a server hosted remotely, you might want to be using [Docker](https://www.docker.com/) containers. Beforehand, you will need to set a few environment variables either manually or by writing an `.env` file in the root directory of this project, like in the example below:
+With the previously described tables, here are all the steps to send a wildfire alert:
+- Prerequisites (ask the instance administrator): register user
+- Register a device: declare your device on the API, using your new user credentials.
+- Create a media object & upload content: using the device credentials, save the picture metadata and upload the image content.
```

Comment:
Do you think it would be needed to explain that there is service to store images (s3 alike) and other tables are in an postgres table ?

Reply:
This is explained in CONTRIBUTING (I think we should scare people only coming for usage / light development rather than "hardcore" developers)